rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -303,15 +316,19 @@ def _decode_repay(
                     resolved_asset = event.asset.resolve_to_asset_with_symbol()
                     event.counterparty = self.counterparty
                     event.notes = f'Return {event.balance.amount} {resolved_asset.symbol} to {self.label}'  # noqa: E501
+                    return_event = event
```

Comment:
here the two events are of type spend. Is it needed the order for the accounting rules? If not maybe you can just return None, None and avoid tracking them

Reply:
it is to avoid any other event (like approval) coming in between and keeping it consistent with others, so that token movements are shown in the last.